enricher/reachability/rust/rust.go

```diff
@@ -69,11 +68,20 @@ func (*Enricher) RequiredPlugins() []string {
 
 // New returns a new rust reachability enricher.
 func New() enricher.Enricher {
```

Comment:
New needs to take in a config (see govulncheck enricher for example) (no need to use the config at the moment, but it does need one. Can be used to configure how the rust proj is compiled in the future for example)